src/Workload/Microsoft.Maui.Dependencies/Microsoft.Maui.Dependencies.csproj

```diff
@@ -25,11 +25,13 @@
     <PackageReference Include="Xamarin.AndroidX.Navigation.Common" />
     <PackageReference Include="Xamarin.AndroidX.Browser" />
     <PackageReference Include="Xamarin.Google.Android.Material" />
+    <PackageReference Include="Xamarin.AndroidX.Security.SecurityCrypto" />
+    <PackageReference Include="Xamarin.Google.Crypto.Tink.Android" />
```

Comment:
Should we consider that IOS still have this bug?

Reply:
Hi, @canperk - what's ths issue you're seeing on iOS? Is it also the same error regarding `Xamarin.AndroidX.Security.SecurityCrypto` ?

Reply:
@rachelkang in case this is still unclear - we are using SecureStorage with a Maui-Blazor app (VS 17.1.0 Preview 2.0) and it works well with iOS (real devices), while Android has the mentioned SecurityCrypto issue (is the fix available in the next update?).

Reply:
This isn't in 17.1 Preview 3 yet. The version after that should have it.